src/app/Login/LoginForm.tsx

```diff
@@ -0,0 +1,122 @@
+"use client";
+import { Input, Button, Link, useDisclosure } from "@nextui-org/react";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { loginValidation } from "@/validation/loginValidation";
+import { onLoginSubmitService } from "./loginService";
+import { useRouter } from "next/navigation";
+import toast from "react-hot-toast";
+import ModalLogin from "@/components/Modal";
+import { useState } from "react";
+
+export default function LoginForm() {
+
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [state, setState] = useState(false);
+  function estado() {
+    setState(true);
+    onOpen();
+  }
+  function estado2() {
+    setState(false);
+    onOpen();
+  }
```

Comment:
para q son estos estados?

Reply:
esos estados son del modal Privacy Policy y Terms of Service, que son el mismo modal pero con el estado le cambio el contenido